content/1-introduction-3-problems.md

```diff
@@ -0,0 +1,129 @@
+Are there domains that involve both imitation and non-copying communication in important degrees?
+How do the two processes interact in such cases?
+These are the questions left open by recognising a complementarity between the Parisian and Californian approaches.
+
+
+### Interaction of cultural and genetic evolution
+
+Opinions on the importance of gene-culture co-evolution dynamics vary widely.
+To what extent do genetic and cultural evolution interact, and in which cases does cultural change drive genetic change?
+Niche construction theorists, and more broadly the Extended Evolutionary Synthesis movement, have convincingly argued for the relevance of such interactions in some specific cases, such as lactose tolerance.
+Developmental systems theorists further argue that genetic material and culture in its broadest sense are part of a wider matrix of resources that participate in the growth of developmental systems, and are thus necessarily interacting.
+Proponents of the Californian approach to cultural evolution broadly agree with the niche construction perspective, considering that the cultural processes they study (concerned more with norms than with traditions *per se*) can have a long-term impact on the selective niche in which later generations develop.
+In particular, one view has it that the centrality of cultural transmission in human beings created a selective pressure for transmission capacities themselves [@sterelny_cultural_2017], such that cultural evolution drives genetic changes that enhance transmission.
+While CAT is broadly compatible with a gene-culture co-evolution account, and also discusses cases of change in context with downstream effects, @morin_how_2016 is not convinced that an impact of culture on genetic change is necessary to explain the emergence of global human traditions.
+Indeed, he argues that faithful transmission itself is neither necessary nor sufficient to explain the diffusion of global traditions, and consequently that scenarios that do not involve genetic adaptation to imitation are just as plausible given the current evidence.
+
+This question is intimately related to the relative importance of imitation and non-copying communication.
+But answers might also lie in the importance and exact nature of the cognitive niche described by @sterelny_minds:_2010:
+how does the construction of such niches contribute to both ecological and psychological (through scaffolding of cognitive development) factors of attraction as the Parisian stream views them?
+How can the process of niche construction be usefully modelled to test hypotheses on the feedback dynamics between different channels of inheritance?
+Here too much is open to explore, which leads us to the two next points:
+formalisation and further development of empirical studies.
+
+
+### Framework versus formal theory contrasted with alternatives
```

Comment:
Multi-level pourrait être mieux dit comme "multi-scale". Par exemple pour une phrase (type Gistr), je pensais faire au fait qu'à chaque échelle il y a beaucoup de dimensions : pour les mots il y a toutes les features possibles, pour la structure il y a les dépendances et la syntaxe, pour le sens il y a le sens intrinsèque, la pragmatique et les implicatures, etc.

Ça vaudrait le coup de développer ? (J'y suis pas attaché)

TODO
- [ ] develop my question here
- [ ] make the title more explicit
- [ ] develop "contender for the detailed explanation of ostensive modes of communication"
- [ ] develop "developing an abstract but systematic mapping of communicative process to stylised phenomenon"